src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicPlatformDetectionTests.cs

```diff
@@ -57,8 +57,6 @@ public async Task SupportedLinuxPlatformsWithMsQuic_IsSupportedIsTrue()
             }
         }
 
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/82154", typeof(PlatformDetection), nameof(PlatformDetection.IsPpc64leProcess))]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/82154", typeof(PlatformDetection), nameof(PlatformDetection.IsUbuntu2004), nameof(PlatformDetection.IsS390xProcess))]
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsInHelix))]
```

Comment:
Note for myself: We are not testing Ppc64 and S390x in Helix anymore. This condition will ensure that this test is disabled for Ppc64 and S390x outside of Helix.